test/e2e/README.md

```diff
@@ -0,0 +1,19 @@
+# E2E Testing
+
+This suite tests the end-to-end operation of the open-feature-operator.
+
+Tests are written with [kuttl](https://kuttl.dev/) and assertions are executed from a curl enabled Job.
+Ngnix reverse proxy is used as the workload where flagd get injected using OFO annotations.
+
+## Running and validating locally
+
+It is recommended to run and validate e2e test locally before opening a pull request.
+
+To run locally (commands are executed from the project root level),
+
+1. Build the operator locally - `docker build . -t open-feature-operator-local:validate`
+2. Create a kind - `kind create cluster --config ./test/e2e/kind-cluster.yml`
+3. Load locally build operator image - `kind load docker-image open-feature-operator-local:validate --name kind`
+4. Deploy Operator to kind cluster - `IMG=open-feature-operator-local:validate make deploy-operator`
+5. Execute kuttl tests - `IMG=open-feature-operator-local:validate make e2e-test-kuttl`
+
```

Comment:
Here I suggest to also mention how a single test case can be executed using the `--test` argument, e.g. `kubectl kuttl test --start-kind=false ./test/testcertificate/ --config=kuttl-test.yaml --test=flagd-disabled`

Reply:
Added document section - https://github.com/open-feature/open-feature-operator/pull/521/commits/c6dbda979c69e9d8931efc7696f513346652bef0#diff-7d9a55ec9712c1ccc931efe126dbea316dab465e6ab265abd089736be5511f47R37-R41